### ISSUE TITLE
Option to not convert link to image

### ISSUE THREAD

**Issue by DonSinDRom** [RocketChat/Rocket.Chat#274]
If someone post a big gif (or other image), client may start lagging.
It will be nice to have ability to switch off/on link->img feature.
I see it as a checkbox under textarea.
Checkbox to switch off/on audio notification will be great too.

**Comment by marceloschmidt** [comment]
Depends on #222

**Comment by gmsecrieru** [comment]
Closing this one as you can disable downloading images automatically from Account -> Preferences
